source/features/git-checkout-pr.tsx

```diff
@@ -125,5 +125,6 @@ void features.add(import.meta.url, {
 	exclude: [
 		pageDetect.isClosedPR,
 	],
+	awaitDomReady: false,
```

Comment:
There are a couple of features that have DOM-based filters AND don't await dom-ready. I only kept those that aren't fundamental, like this exclusion. I prefer a faster feature to a console without errors. This can be fixed in a better way (e.g. move this check inside the event listener)

Same goes for features that use `isBlank` as an exclusion: as mentioned in #5222, the manager should instead catch the error and silence it on blank pages.